src/main/java/appeng/util/item/OreHelper.java

```diff
@@ -106,10 +87,12 @@ public OreReference isOre( final ItemStack itemStack )
 
 	boolean sameOre( final AEItemStack aeItemStack, final IAEItemStack is )
 	{
-		final OreReference a = aeItemStack.getDefinition().getIsOre();
```

Comment:
Am I blind or was that always returning true?

Reply:
This method don't see any real usage in ae2 code base, so 🤷‍♂️

Reply:
It is used by Platform.extractItemsByRecipe(). You probably mixed it with another one that is used only by AEItemStack.fuzzyComparison, which is in turn unused in AE2 itself (but is part of public API)